src/History.js

```diff
@@ -186,6 +186,7 @@ var EnyoHistory = module.exports = kind.singleton(
 		_history.splice(0, _history.length);
 		_popQueueCount = 0;
 		_pushQueued = false;
+		this.stopJob('history.go');
```

Comment:
This just becomes the same as enabledChanged(). I wonder if enabledChanged should simply call this.clear() instead and add _processing = false;